packages/share_plus/share_plus_web/lib/share_plus_web.dart

```diff
@@ -61,15 +61,11 @@ class SharePlusPlugin extends SharePlatform {
     String? subject,
     String? text,
     Rect? sharePositionOrigin,
-  }) {
+        ShareWithAppWindows? appName
```

Comment:
should be ShareWithApp?

Reply:
yes, I fixed all above issues in latest commits